BaseTools/Conf/build_rule.template

```diff
@@ -247,50 +260,6 @@
         Trim --trim-long --source-code -o ${d_path}(+)${s_base}.iii ${d_path}(+)${s_base}.ii
         "$(NASM)" -I${s_path}(+) $(NASM_INC) $(NASM_FLAGS) -o $dst ${d_path}(+)${s_base}.iii
 
-[Rust-To-Lib-File]

```

Comment:
This change should not be touching Rust related support.

Reply:
Fixed with some clean cherry-picking.